src/parser/data/LimitOffsetClause.h

```diff
@@ -57,5 +59,11 @@ struct LimitOffsetClause {
   // almost always.
   bool isUnconstrained() const { return !_limit.has_value() && _offset == 0; }
 
+  // Return true if and only if it is OK to process the query lazily.
+  //
+  // NOTE: If we don't modify the `LIMIT` of the original query, this is always
+  // fine.
+  bool requestLaziness() const { return true; }
```

Comment:
I'm not convinced it makes sense to have this as a function of this struct if it always returns true. It suggests that the result is dependent on the values when in fact it's just a constant.

Reply:
It was just there for testing and playing around. I also don't see a use for it anymore and will remove it.